curriculum/challenges/english/03-front-end-libraries/react/use--for-a-more-concise-conditional.md

```diff
@@ -10,7 +10,9 @@ dashedName: use--for-a-more-concise-conditional
 
 The `if/else` statements worked in the last challenge, but there's a more concise way to achieve the same result. Imagine that you are tracking several conditions in a component and you want different elements to render depending on each of these conditions. If you write a lot of `else if` statements to return slightly different UIs, you may repeat code which leaves room for error. Instead, you can use the `&&` logical operator to perform conditional logic in a more concise way. This is possible because you want to check if a condition is `true`, and if it is, return some markup. Here's an example:
 
-`{condition && <p>markup</p>}`
+```js
+{condition && <p>markup</p>}
+```
```

Comment:
```suggestion
```jsx
{condition && <p>markup</p>}
```
```
Mirroring @RandellDawson's suggestion from the previous PR.